R/access-element.R

```diff
@@ -0,0 +1,44 @@
+#' @title Add Access Element
+#' @description The add access element creates a list that includes all the required
+#' elements for the access section of an EML. Calling add_access() will output the 
+#' default public access permissions. To create a non default access add principal 
+#' and permission values into allow_principal and add_permission. 
+#' @param allow_principal The principal argument refers to who will have access
+#' to this dataset. The allow_principal argument takes in a string. 
+#' With no input the allow_principal will default to "public". To change
```

Comment:
please note the other valid options for principal and permission?

Reply:
Added.